registry/storage/driver/s3-aws/s3.go

```diff
@@ -243,6 +250,28 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		return nil, fmt.Errorf("The secure parameter should be a boolean")
 	}
 
+	skipVerifyBool := false
+	skipVerify := parameters["skipverify"]
```

Comment:
Would make this optional and default to `false` to be compatible with older registry configs.

Reply:
Yeah, the parsing function below this block will ignore unset values, which would make `skipVerifyBool` default to false.

Reply:
Oh yup. Disregard then.